### ISSUE TITLE
Interested in Contributions?

### ISSUE THREAD

**Issue by engineerjoe440** [TheStaticTurtle/pyVBAN#6]
Hi there! Awesome project! :tada: 

I've been poking around, and found a few ways that I think I could help offer some enhancements. Are you open to contributions?

**Comment by TheStaticTurtle** [comment]
Hi, thanks for the interest in the project, as I said in the other issue, it's a very old project. It's definitely not up to the current [VBAN specs](https://vb-audio.com/Voicemeeter/VBANProtocol_Specifications.pdf) (now that I think of it, it probably wasn't when I wrote it).

It's certainly in a state where I wouldn't push it to its own repo now. I would most likely do a complete rewrite if I needed something similar again 😅 

You're welcome to submit PRs if you like. What did you want to do/add?

**Comment by engineerjoe440** [comment]
I was mostly looking at adding some to a few of the docstrings, and adding some suggestions regarding the `PyAudio` exception handling. I did some playing with it, yesterday, and learned a few things about how PyAudio throws exceptions unless disabled. I also found a way that we could make the transmitter do a "restart" to reconnect the stream when exceptions are thrown.

I forked and added some of those changes, so I'll throw up a draft PR for you, just to take a look at!

Thanks again! This is slick!
